registry/repository.go

```diff
@@ -143,5 +190,53 @@ func (c *repositoryClient) uploadSignatureManifest(ctx context.Context, subject,
 		PackImageManifest:   c.OCIImageManifest,
 	}
 
-	return oras.Pack(ctx, c.Repository, ArtifactTypeNotation, []ocispec.Descriptor{blobDesc}, opts)
+	return oras.Pack(ctx, c.GraphTarget, ArtifactTypeNotation, []ocispec.Descriptor{blobDesc}, opts)
+}
+
+// predecessors returns predecessor nodes of desc in target and filter by artifactType
+func predecessors(ctx context.Context, target content.ReadOnlyGraphStorage, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
```

Comment:
This function should be named as `signatureReferrers`. The documentation is required to be updated accordingly.